touchscreen.lua

```diff
@@ -156,7 +156,7 @@ local function create_formspec(meta, data)
 	if meta:get_int("no_prepend") == 1 then
 		fs = fs.."no_prepend[]"
 	end
-	if not meta:get("real_coordinates") and not meta:get("realcoordinates") then
+	if meta:get_int("real_coordinates") == 0 then
```

Comment:
The old metadata value should also be checked, for compatibility with old touchscreens.
```suggestion
	if meta:get_int("real_coordinates") == 0 and meta:get_int("realcoordinates") == 0 then
```
This would cause the same problem for old touchscreens, but I think that's better than ignoring it completely.

Reply:
oh, somehow I missed that old screens used a different key to store this setting. Thanks, I added a fallback to the old metadata value.